### PR TITLE
setup for Kommander 1.4, beta 0

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+navigationTitle: Kommander v1.4 Release Notes
+title: Release Notes
+menuWeight: 0
+excerpt: View release-specific information for Kommander
+enterprise: false
+---
+
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Kommander&reg; version 1.4.0 Beta 0 was released on ?????.**
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:13]:
```suggestion
**D2iQ&reg; Kommander&reg; version 1.4.0 Beta 0 was released on March 10, 2021.**
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Rate limiting happens on a per-pull basis regardless of whether the pulled image is owned by a paid user. This means D2iQ, as owner of most images used in Konvoy, does not have any influence as to whether your current address is rate-limited or not. Konvoy does not have a strict dependency on Docker Hub accounts or plans.
+
+For more information on addressing this limit, see [Docker hub rate limits](../operations/manage-docker-hub-rate-limits).
+
+# New Features
+## Feature #1 
+Customers now .
+
+For more information, see .
+## Component versions
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:45]: Are these the latest version numbers?

**Review by russurquhart1 — APPROVED** [review]

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Supported Versions
+| Kubernetes Support | Version |
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:25]:
```suggestion
Kubernetes Support | Version
-- | --
Minimum | 1.18.0
Maximum | 1.20.x
Default | 1.20.2


```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:27]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+| **Maximum**        | 1.19.x  |
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:28]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+| **Default**        | 1.19.7  |
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:29]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+For more information on addressing this limit, see [Docker hub rate limits](../operations/manage-docker-hub-rate-limits).
+
+# New Features
+## Feature #1 
+Customers now .
+
+For more information, see .
+## Component versions
+- Addon: 1.4.0-6
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:46]:
```suggestion
- Addon: 1.4.0-1
- Chart: 0.16.1
- kommander-federation (yakcl): 0.9.0
- kommander-licensing (yakcl): 0.9.0
- UI: 6.88.0
- kommander-karma: 0.3.12
- kubeaddons-catalog: 0.1.15
- kommander-thanos: 0.1.16
- kubecost: 0.5.4
- grafana: 6.6.0
- karma: 0.70
- thanos: 0.10.1
- cost-analyzer: 1.71.1
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+- Chart: 0.13.7
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:47]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+- kommander-federation (yakcl): 0.7.0
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:48]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+- kommander-licensing (yakcl): 0.7.0
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:49]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+- UI: 6.70.0
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:50]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+- kommander-karma: 0.3.12
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:51]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+- kubeaddons-catalog: 0.1.15
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:52]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+- kommander-thanos: 0.1.16
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:53]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+- kubecost: 0.1.15
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:54]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+- grafana: 6.6.0
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:55]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 17]
```diff
@@ -0,0 +1,60 @@
...
+- karma: 0.70
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:56]:
```suggestion
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 18]
```diff
@@ -0,0 +1,60 @@
...
+- thanos: 0.10.1
+- cost-analyzer: 1.70.1
```
Comment (russurquhart1) [inline pages/dkp/kommander/1.4/release-notes/index.md:58]:
```suggestion
```

**Review by russurquhart1 — APPROVED** [review]

File: pages/dkp/kommander/1.4/release-notes/index.md (hunk 1 above)
Comment (gracedo) [reply pages/dkp/kommander/1.4/release-notes/index.md:13]: 3/10 is the kommander 1.3.1 patch release. This 1.4.0 beta has already been released, last friday with konvoy 1.8.0 beta

Comment (gracedo) [reply pages/dkp/kommander/1.4/release-notes/index.md:13]:
```suggestion
**D2iQ&reg; Kommander&reg; version 1.4.0 Beta 0 was released on 26, February 2021.**
```

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 19]
```diff
@@ -0,0 +1,60 @@
...
+## Docker hub rate limiting 
+Docker Hub announced an update to their image pull policies in August, 2020. The change results in the need to change cluster configurations to accommodate new account structures that enable image pull rate limiting.
+
+Rate limiting happens on a per-pull basis regardless of whether the pulled image is owned by a paid user. This means D2iQ, as owner of most images used in Konvoy, does not have any influence as to whether your current address is rate-limited or not. Konvoy does not have a strict dependency on Docker Hub accounts or plans.
+
+For more information on addressing this limit, see [Docker hub rate limits](../operations/manage-docker-hub-rate-limits).
+
+# New Features
+## Feature #1 
+Customers now .
```
[the remark this answers is not in the document]
Comment (gracedo) [reply pages/dkp/kommander/1.4/release-notes/index.md:42]: can probably remove Feature #1 etc, haven't shipped any feats in the beta yet

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 20]
```diff
@@ -0,0 +1,60 @@
...
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
+| **Maximum**        | 1.19.x  |
+| **Default**        | 1.19.7  |
+
+# Breaking changes
+
+## Docker hub rate limiting 
+Docker Hub announced an update to their image pull policies in August, 2020. The change results in the need to change cluster configurations to accommodate new account structures that enable image pull rate limiting.
```
Comment (gracedo) [inline pages/dkp/kommander/1.4/release-notes/index.md:34]: I think this was copied from 1.3 and doesn't apply to 1.4 anymore

Comment (cmastr) [reply pages/dkp/kommander/1.4/release-notes/index.md:34]: it does still apply.

**Pull request merged**